Jenkinsfile

```diff
@@ -39,6 +39,13 @@ pipeline {
             ((Get-Content -path ./appsettings.Production.json -Raw) -replace '<REPLACEME:ntbsContext>',"$env:INT_DB_CONNECTION_STRING") | Set-Content -Path ./appsettings.Production.json
           ''')
         }
+        withCredentials([string(credentialsId: 'int-audit-connection-string', variable: 'INT_AUDIT_CONNECTION_STRING')]) {
```

Comment:
I've added this in line with what was there for the normal db string - where are these secrets defined, and how can I update them? What about the other environments?

Reply:
Oh, that file has changed lots since 😂 Sorry!! So this is actually no longer needed, as you'll notice in the merge.
We're giving deployments the right connection string by setting env variables in the form of `ConnectionsStrings__<context>=<value>`. This is defined in https://github.com/PublicHealthEngland/ntbs_Beta/blob/master/ntbs-service/int.yml#L18 (and in test.yml and research.yml files)
The secret can be defined on the kubernetes cluster through cmd line as per https://github.com/PublicHealthEngland/ntbs_Beta/tree/master/ntbs-service#maintanance